tests/distributions/test_continuous.py

```diff
@@ -432,18 +432,43 @@ def test_exponential(self):
             {"lam": Rplus},
             lambda value, lam: st.expon.logpdf(value, 0, 1 / lam),
         )
+        check_logp(
```

Comment:
No need to check logp and logcdf, the new random test below is enough.

Reply:
Okay got it! Will remove them in next commit